doc/installing.md

```diff
@@ -304,6 +304,21 @@ sudo -E PATH=${PATH} LD_LIBRARY_PATH=${LD_LIBRARY_PATH} $(go env GOPATH)/bin/lxd
 If `newuidmap/newgidmap` tools are present on your system and `/etc/subuid`, `etc/subgid` exist, they must be configured to allow the root user a contiguous range of at least 10M UID/GID.
 ```
 
+### Shell completions
+
+Shell completion profiles can be generated with `lxc completion <shell>` (e.g. `lxc completion bash`). Supported shells are `bash`, `zsh`, `fish`, and `powershell`.
+
+```sh
+lxc completion <shell> > /tmp/completions
+. /tmp/completions
+```
+
+If you are using `bash-completion`:
+
+```bash
+lxc completion bash > /etc/bash_completion.d/lxc
+```
```

Comment:
I'd probably just give the `bash` example for brevity. So just:

```bash
lxc completion bash > /etc/bash_completion.d/lxc
. /etc/bash_completion.d/lxc
```

Sorry for not mentioning this on the first review pass.